app/services/slack_notification_service.rb

```diff
@@ -16,9 +21,44 @@ def send_notification
 
   private
 
-  def notify_open_pull_request
+  def notify_pull_request
     pull_request_url = extra_params[:pull_request][:html_url]
-    client = Slack::Web::Client.new
-    client.chat_postMessage(channel: '#code-review', text: pull_request_url)
+    client.chat_postMessage(channel: CHANNEL, text: pull_request_url)
+  end
+
+  def filter_unlabeled_action
+    label_name = extra_params[:github_webhook][:label][:name]
+    notify_pull_request if on_hold? label_name
+  end
+
+  def filter_labeled_action
```

Comment:
I think the following code is more readable and more performant (micro-perfomance)
```
def filter_labeled_action
  label_name = extra_params[:github_webhook][:label][:name]
  return unless on_hold?(label_name)

  matches = find_message pull_request_url
  delete_message matches
end
```

Additionally, I would call this method `filter_on_hold_labeled_action`.